### PR TITLE
Add parameter description for 2012 version

### PR DESCRIPTION
**Pull request by velkovb**
Added parameter description for 2012 version of the article and adjusted formatting of all version so it is the same everywhere.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Change looks good.

File: docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md [hunk 1]
```diff
@@ -135,7 +156,14 @@ Accept wildcard characters: False
 ```
 
 ### -DeniedList
-{{Fill DeniedList Description}}
+Specifies the users, computers, groups or other accounts to add to the list of accounts that are denied the right to replicate their passwords to this RODC.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
Specifies the users, computers, groups, or other accounts to add to the list of accounts that are denied the right to replicate their passwords to this RODC.
```

File: docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md [hunk 2]
```diff
@@ -180,7 +227,27 @@ Accept wildcard characters: False
 ```
 
 ### -Server
-{{Fill Server Description}}
+Specifies the Active Directory Domain Services instance to connect to, by providing one of the following values for a corresponding domain name or directory server.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
Specifies the Active Directory Domain Services instance to connect to by providing one of the following values for a corresponding domain name or directory server.
```

File: docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md [hunk 3]
```diff
@@ -180,7 +227,27 @@ Accept wildcard characters: False
...
+The service may be any of the following: Active Directory Lightweight Domain Services, Active Directory Domain Services or Active Directory snapshot instance.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
The service may be any of the following: Active Directory Lightweight Domain Services, Active Directory Domain Services, or Active Directory snapshot instance.
```

File: docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md [hunk 4]
```diff
@@ -211,28 +278,24 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Remove-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

File: docset/winserver2012r2-ps/addsadministration/Remove-ADDomainControllerPasswordReplicationPolicy.md [hunk 5]
```diff
@@ -286,24 +286,24 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/addsadministration/Remove-ADDomainControllerPasswordReplicationPolicy.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by e0i** [comment]
@get-itips Any additional input?

**Comment by traya1** [comment]
Hi @e0i - We understand that these [PR criteria](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master) are new, and we appreciate your patience with the process. 

**Following are blocking issues that need to be resolved before we can merge:**

docset/windows/addsadministration/remove-addomaincontrollerpasswordreplicationpolicy.md
- merge conflict

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)
- The PR is free of merge conflicts. If a merge conflict exists, then see [Resolve merge conflicts in Git and GitHub](https://review.docs.microsoft.com/en-us/help/contribute/resolve-merge-conflicts?branch=master).

***

Thanks

**Comment by e0i** [comment]
Merge conflict(s) have been resolved. Thanks.

**Pull request merged**